src/Raven.Server/Documents/Indexes/Static/JavaScriptIndex.cs

```diff
@@ -54,6 +54,14 @@ public JavaScriptIndex(IndexDefinition definition)
                 }
                 list.Add(chainList);
             }
+
+            var reduce = definitions.GetProperty("reduce").Value;
+            if (reduce.IsNull() == false)
+            {
+                GroupByFields = new string[] { };
+                Reduce = items => JintReduceFuncWrapper(reduce, items);
+                Console.WriteLine("Not NULL!");
```

Comment:
Should remove this

Reply:
This is debugging leftover, the code is still in early stages of WIP